lib/actions/api.js

```diff
@@ -548,20 +548,30 @@ const findStopTimesForStopError = createAction('FIND_STOP_TIMES_FOR_STOP_ERROR')
  */
 export function findStopTimesForStop (params) {
   return function (dispatch, getState) {
-    let { stopId, ...otherParams } = params
+    const { date, stopId, ...otherParams } = params
+    let datePath = ''
+    if (date) {
+      const dateWithoutDashes = date.replace(/-/g, '')
+      datePath = `/${dateWithoutDashes}`
+    }
+
     // If other params not provided, fall back on defaults from stop viewer config.
-    const queryParams = { ...getStopViewerConfig(getState().otp), ...otherParams }
-    // If no start time is provided, pass in the current time. Note: this is not
+    // const queryParams = { ...getStopViewerConfig(getState().otp), ...otherParams }
+    // TODO: parform an additional query with ^^ and without date path if the one below returns no stop times at all.
```

Comment:
I don't recall what this stopViewerConfig stuff used to do. I am noticing, however, that this PR has changed the behavior of the "live" stop viewer. We should be showing "extra" stop times even if they are not today. I'm not really sure if this is related to fixes you tried to make to address #287 or if it's related to something else.

Using a TriMet config, navigate to http://localhost:9966/#/stop/TriMet:4105 and compare with https://newplanner.trimet.org/map/#/stop/TriMet:4105

## localhost:
![image](https://user-images.githubusercontent.com/2370911/102521045-6cb97c00-4062-11eb-85cb-b5d926e469c0.png)
## newplanner:
![image](https://user-images.githubusercontent.com/2370911/102521085-7ba02e80-4062-11eb-8f27-48382c696b6f.png)

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/fac518e21590d006975dd444f26a0c07d7de7328.